go.mod

```diff
@@ -4,11 +4,13 @@ go 1.18
 
 require (
 	github.com/hashicorp/terraform-plugin-sdk v1.17.2
-	github.com/opslevel/opslevel-go/v2022 v2022.10.22
+	//github.com/opslevel/opslevel-go/v2022 v2022.10.22
 	github.com/rs/zerolog v1.28.0
 	github.com/shurcooL/graphql v0.0.0-20220606043923-3cf50f8a0a29
 )
 
+require github.com/opslevel/opslevel-go/v2022 v2022.0.0-00010101000000-000000000000
```

Comment:
this might be part of the reasons - you shouldn't need to do this - just update the submodule and uncomment the lines at the bottom of this file

Reply:
Yeah, fixing that now. It was uncommented and goland was still acting weird. Seems to be fine now so I'll push this change.